### PR TITLE
[16.0][IMP] account_reconcile_oca: Open reconcile view when press the name on journal name

### PR DESCRIPTION
**Pull request by etobella**
Similar to Odoo EE functionality

With the change, we open directly reconcile view when pressing the open_action button

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Can you put a GIF or screenshot about the change?

**Comment by brambuijs** [comment]
@etobella This would be very usefull. Reconcilation would be within one button click from the dashboard. This pr from @ByteMeAsap https://github.com/OCA/account-reconcile/pull/710 because being able to access the statements in one click is usefull as well

File: account_reconcile_oca/models/account_journal.py [hunk 1]
```diff
@@ -31,3 +31,11 @@ def get_rainbowman_message(self):
         if self.get_journal_dashboard_datas()["number_to_reconcile"] > 0:
             return False
         return _("Well done! Everything has been reconciled")
+
+    def open_action(self):
+        self.ensure_one()
+        if self.type not in ["bank", "cash"]:
+            return super().open_action()
+        return self.env["ir.actions.actions"]._for_xml_id(
+            "account_reconcile_oca.action_bank_statement_line_open"
```
Comment (chienandalu) [inline account_reconcile_oca/models/account_journal.py:40]: This action is the same as the new one you defined
```suggestion
            "account_reconcile_oca.action_bank_statement_line_reconcile_all"
```

File: account_reconcile_oca/views/account_bank_statement_line.xml [hunk 2]
```diff
@@ -349,6 +349,26 @@
...
+            eval="[(5, 0, 0),
+            (0, 0, {'view_mode': 'kanban', 'view_id': ref('bank_statement_line_reconcile_view')}),
+            (0, 0, {'view_mode': 'tree', 'view_id': ref('account_statement_base.account_bank_statement_line_tree')})]"
+        />
+        <field name="help" type="html">
+            <p class="o_view_nocontent_smiling_face">
+            Nothing to reconcile
+            </p>
+        </field>
+    </record>
```
Comment (chienandalu) [inline account_reconcile_oca/views/account_bank_statement_line.xml:371]: `account_reconcile_oca.action_bank_statement_line_reconcile_all` is the same action
```suggestion
```

**Review by chienandalu — CHANGES_REQUESTED** [review]
I agree with the ux concept. Just some implementation remarks :point_down:

**Comment by chienandalu** [comment]
@etobella did you check my comments

**Review by brambuijs — APPROVED** [review]
Great addition see my previous comment

**Comment by pedrobaeza** [comment]
Superseded by #763

**Pull request closed without merging**